app/grandchallenge/charts/specs.py

```diff
@@ -204,9 +204,9 @@ def components_line(*, values, title, cpu_limit, tooltip):
                 ],
                 "encoding": {
                     "x": {
-                        "timeUnit": "hoursminutesseconds",
+                        "timeUnit": "yearmonthdatehoursminutesseconds",
                         "field": "Timestamp",
-                        "title": "Local Time / HH:MM:SS",
+                        "title": "Local Date/Time - MMM DD, YYYY HH:MM:SS",
```

Comment:
People can probably work it out now, same goes for the instances of this.

```suggestion
                        "title": "Local Time",
```